web example panic

Any pointers would be helpful.  I was trying to run the web example, and have the following environment:

Go 1.6.2 on MacOSX 10.11.5
```
~/.stormpath/stormpath.yaml
---
stormpath:
  client:
    apiKey:
      id: "XXXXX"
      secret: "YYYYY"
  application:
    name: "GSK"
    href: "https://api.stormpath.com/v1/applications/2qkEiSjJBR3cE8ALRnBxgg"
```

Steps:

`cd web/example`
`go run example.go`

Output:

```
ERROR: 2016/06/17 11:16:26 viper.go:1034: svType != tvType; key=name, st=string, tt=<nil>, sv=GSK, tv=<nil>
ERROR: 2016/06/17 11:16:26 viper.go:1034: svType != tvType; key=href, st=string, tt=<nil>, sv=https://api.stormpath.com/v1/applications/2qkEiSjJBR3cE8ALRnBxgg, tv=<nil>
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x8 pc=0x12ad68]

goroutine 1 [running]:
panic(0x552260, 0xc82000a100)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/jarias/stormpath-sdk-go/web.resolveAccountStores(0xc8204a0640)
	/Users/stanford/devel/go_workspace/src/github.com/jarias/stormpath-sdk-go/web/web.go:110 +0x1c8
github.com/jarias/stormpath-sdk-go/web.NewStormpathMiddleware(0xd54348, 0xc820112ab0, 0xc8201108e0, 0x1, 0x1, 0x0)
	/Users/stanford/devel/go_workspace/src/github.com/jarias/stormpath-sdk-go/web/web.go:77 +0x18a
main.main()
	/Users/stanford/devel/go_workspace/src/github.com/jarias/stormpath-sdk-go/web/example/example.go:60 +0x141
exit status 2
```

--- a/web/web.go
+++ b/web/web.go
@@ -107,7 +107,7 @@ func resolveAccountStores(application *stormpath.Application) {
 		panic(fmt.Errorf("No account stores are mapped to the specified application. Account stores are required for login and registration. \n"))
 	}
 
-	if application.DefaultAccountStoreMapping.Href == "" && Config.RegisterEnabled {
+	if application.DefaultAccountStoreMapping == nil && Config.RegisterEnabled {
 		panic(fmt.Errorf("No default account store is mapped to the specified application. A default account store is required for registration. \n"))
 	}
 }

